Tasks/Common/nuget-task-common/module.json

```diff
@@ -2,6 +2,7 @@
     "messages": {
         "NGCommon_IgnoringNuGetExtensionsPath": "Detected NuGet extensions loader path (NUGET_EXTENSIONS_PATH environment variable). Extensions are ignored when using the built-in NuGet client",
         "NGCommon_DetectedNuGetExtensionsPath": "Detected NuGet extensions loader path (NUGET_EXTENSIONS_PATH environment variable): %s",
-        "NGCommon_UnableToFindTool": "Unable to find tool %s"
+        "NGCommon_UnableToFindTool": "Unable to find tool %s",
+        "NGCommon_UnabletoDetectNuGetVersion" : "Unknown NuGet version selected."
```

Comment:
You'll need to duplicate this message into the individual task.json files as well, unfortunately